docs/equinix-metal.md

```diff
@@ -10,7 +10,7 @@ dynamically from the Terraform state file.
 ## Local Host Configuration
 
 To perform this installation, you will need a localhost to run Terraform/Ansible (laptop, VM, etc) and an account with Equinix Metal.
-In this example, we're using an m1.large CentOS 7 OpenStack VM as the localhost to kickoff the Kubernetes installation.
+In this example, we're using a m1.large CentOS 7 OpenStack VM as the localhost to kick off the Kubernetes installation.
```

Comment:
The original seems good for me.

Reply:
Fine by me for the `kick off` part, kind of fine by me for the `a m1.large Centos 7 VM` instead of `an` but not sure there is really a rule there so up to you.